esmvalcore/cmor/check.py

```diff
@@ -539,6 +541,26 @@ def _check_coord_values(self, coord_info, coord, var_name):
                 self._cube.remove_coord(coord)
                 self._cube.add_aux_coord(new_coord, dims)
 
+    def _check_longitude_max(self, coord, var_name):
+        if not self.automatic_fixes:
+            return False
```

Comment:
```suggestion
```

Reply:
This code will never be executed anyway because of the if statement outside this function